packages/react-components/src/atoms/Markdown.tsx

```diff
@@ -15,7 +16,10 @@ interface MarkdownProps {
 }
 
 const Markdown = ({ value, toc = false }: MarkdownProps) => {
-  let processor = unified().use(markdown).use(remark2rehype);
+  let processor = unified()
+    .use(markdown)
+    .use(remark2rehype, { allowDangerousHtml: true })
```

Comment:
How does this work? We are just allowing HTML tags to be injected and the contentful MD editor provides HTML?

Reply:
The contentful MD editor allows entering html. On the frontend, the description field also allows entering html. So wherever an output is created/edited and a <sub> or <sup> tag is included in the description, it should render as a subscript/superscript on the frontend when saved

Reply:
Im not sure this is a good solution. We have been trying to restrict what users can enter into the website while allowing some formatting. I don't think we can rely on sanatize to prevent things like https://stackoverflow.com/a/701609/324989.

Reply:
It does handle the case you referenced but no I can't guarantee that it handles all existing cases. I'll look into supporting the extended MD set instead.